.github/workflows/ipa-release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release IPA Package
+
+on:  
+  workflow_dispatch:
+  push: 
+    branches:
+      - main
+    paths:
+      - 'tools/spectral/ipa/package.json'
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest  
+    
+    steps:
+    - name: Checkout Repository  
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      with:
+          sparse-checkout: |
+            tools/spectral/ipa  
+        
+    - name: Check Version Bump
+      working-directory: tools/spectral/ipa
+      id: version_check
```

Comment:
Inline bash makes is really hard for us to test or verify those actions.
Following [best practices](https://medium.com/@wtr/best-patterns-building-github-actions-workflows-6b03655223ca) consider using npm/bash scripts and call them from action.

This way we can actually execute and verify relase independently from gh actions env